testing/test_aux.sh

```diff
@@ -15,6 +15,23 @@ echo Lint checks | tee -a $TEST_RESULTS
 cmd/inbuild skip
 echo cmd/inbuild exit code $? | tee -a $TEST_RESULTS
 
+tes# mac_oui testing 
+echo Running mac_oui tests  | tee -a $TEST_RESULTS
+cp misc/system_base.conf local/system.conf
+cat <<EOF > local/site/module_config.json
```

Comment:
You can't use a local/ file in the test scripts.

Reply:
We have already discussed this, you told me that suggesting a local/ file in the test scripts was ok, and this was a false alarm. If the situation has changed then could you let me know what should be placed here instead?